src/core/stdc/math.d

```diff
@@ -3500,81 +3500,81 @@ else version(DragonFlyBSD)
-
-    real acoshl(real x);
-    real asinhl(real x);
-    real atanhl(real x);
-    real coshl(real x);
-    real sinhl(real x);
-    real tanhl(real x);
-
-    real expl(real x);
-    real exp2l(real x);
-    real expm1l(real x);
-    real frexpl(real x, int *exp);
+    pure real acosl(real x);
+    pure real asinl(real x);
+    pure real atanl(real x);
+    pure real atan2l(real x, real y);
+    pure real cosl(real x);
+    pure real sinl(real x);
+    pure real tanl(real x);
+
+    pure real acoshl(real x);
+    pure real asinhl(real x);
+    pure real atanhl(real x);
+    pure real coshl(real x);
+    pure real sinhl(real x);
+    pure real tanhl(real x);
+
+    pure real expl(real x);
+    pure real exp2l(real x);
+    pure real expm1l(real x);
+    pure real frexpl(real x, int *exp);
     int ilogbl(real x);
-    real ldexpl(real x, int exp);
+    pure real ldexpl(real x, int exp);
     real logl(real x);
     real log10l(real x);
     real log1pl(real x);
     real log2l(real x);
-    real logbl(real x);
-    real modfl(real x, real *iptr);
-    real scalbnl(real x, int y);
-    real scalblnl(real x, c_long y);
-
-    real cbrtl(real x);
-    real fabsl(real x);
-    real hypotl(real x, real y);
-    real powl(real x, real y);
-    real sqrtl(real x);
-
-    real erfl(real x);
-    real erfcl(real x);
-    real lgammal(real x);
-    real tgammal(real x);
-
-    real ceill(real x);
-    real floorl(real x);
-    real nearbyintl(real x);
-    real rintl(real x);
+    pure real logbl(real x);
+    pure real modfl(real x, real *iptr);
+    pure real scalbnl(real x, int y);
+    pure real scalblnl(real x, c_long y);
+
+    pure real cbrtl(real x);
+    pure real fabsl(real x);
+    pure real hypotl(real x, real y);
+    pure real powl(real x, real y);
+    pure real sqrtl(real x);
+
+    pure real erfl(real x);
+    pure real erfcl(real x);
+    pure real lgammal(real x);
+    pure real tgammal(real x);
+
+    pure real ceill(real x);
+    pure real floorl(real x);
+    pure real nearbyintl(real x);
+    pure real rintl(real x);
     c_long lrintl(real x);
     long llrintl(real x);
-    real roundl(real x);
+    pure real roundl(real x);
     c_long lroundl(real x);
     long llroundl(real x);
-    real truncl(real x);
+    pure real truncl(real x);
 
-    real fmodl(real x, real);
-    real remainderl(real x, real);
-    real remquol(real x, real y, int *iptr);
+    pure real fmodl(real x, real);
+    pure real remainderl(real x, real);
+    pure real remquol(real x, real y, int *iptr);
 
-    real copysignl(real x, real y);
-    real nanl(const char *);
-    real nextafterl(real x, real y);
-    real nexttowardl(real x, real y);
+    pure real copysignl(real x, real y);
+    pure real nanl(const char *);
+    pure real nextafterl(real x, real y);
+    pure real nexttowardl(real x, real y);
 
-    real fdiml(real x, real y);
-    real fmaxl(real x, real y);
-    real fminl(real x, real y);
+    pure real fdiml(real x, real y);
```

Comment:
For non-DragonFlyBSD `fdiml` isn't marked pure. Same for `copysign`, `nan`, `nexttoward`, `nextafter`.